src/connector/ui.jsx

```diff
@@ -253,7 +266,20 @@ Zotero.GoogleDocs.UI = {
 			button2Text: "",
 			message: Zotero.getString('integration_googleDocs_docxAlert', ZOTERO_CONFIG.CLIENT_NAME),
 		};
-		Zotero.Inject.confirm(options);
+		return Zotero.Inject.confirm(options);
+	},
+
+	displayOrphanedCitationAlert: async function() {
+		const options = {
+			title: Zotero.getString('general_warning'),
+			button2Text: "",
+			button3Text: Zotero.getString('general_moreInfo'),
+			message: Zotero.getString('integration_googleDocs_orphanedCitationAlert', ZOTERO_CONFIG.CLIENT_NAME),
+		};
+		let result = await Zotero.Inject.confirm(options);
+		if (result.button == 3) {
+			Zotero.Connector_Browser.openTab('https://www.zotero.org/support/kb/google_docs_citations_unlinked');
```

Comment:
Opening the link should cancel the edit and leave the citation in the document. Otherwise, the citation dialog can appear over the wiki page, and it's also just much harder to figure out what's going on.